### ISSUE TITLE
EXCEPTION_ACCESS_VIOLATION at address 0x005f39f7

### ISSUE THREAD

**Issue by Uveso** [FAForever/fa#1574]
5 Jan 2015
https://github.com/FAForever/fa/issues/325 (from Sheeo)
28 Jan 2015
https://github.com/FAForever/fa/issues/423 (from Crotalus)
19 Feb 2015
https://forums.faforever.com/viewtopic.php?f=3&t=9458
12 Jul 2015
http://forums.faforever.com/viewtopic.php?p=104006
11 Jan 2016
http://forums.faforever.com/viewtopic.php?p=117527
01 Oct 2016
http://forums.faforever.com/viewtopic.php?f=3&t=13243
21.Feb 2017
http://forums.faforever.com/viewtopic.php?f=3&t=14066
31.Mar 2017
http://forums.faforever.com/viewtopic.php?f=3&t=14306
24.Apr 2017
http://forums.faforever.com/viewtopic.php?f=3&t=14422
20.May 2017
http://forums.faforever.com/viewtopic.php?f=3&t=14613
19.Jun 2017
http://forums.faforever.com/viewtopic.php?f=3&t=14772
06.Feb 2018
http://forums.faforever.com/viewtopic.php?f=3&t=15866
22.Apr 2018
http://forums.faforever.com/viewtopic.php?f=3&t=16125
24.Apr 2018
http://forums.faforever.com/viewtopic.php?f=3&t=16133

In summary:
If this error occurs in multiplayer all players crash simultaneously with the same error (0x005f39f7)
Also happens in single player vs AI
No warnings, game just crashes. One player said he was "issuing orders" before the game crashed.
Only a few replies after first Post. Mostly the error don't returned.

No idea so far, what could cause this.

**Comment by aeoncleanse** [comment]
http://forums.faforever.com/viewtopic.php?f=3&t=14066

Another...

**Comment by speed2CZ** [comment]
http://forums.faforever.com/viewtopic.php?f=3&t=14772

ladder game

**Comment by PhilipJFryFAF** [comment]
http://forums.faforever.com/viewtopic.php?f=3&t=15866

**Comment by PhilipJFryFAF** [comment]
http://forums.faforever.com/viewtopic.php?f=3&t=16125

**Comment by PhilipJFryFAF** [comment]
http://forums.faforever.com/viewtopic.php?f=3&t=16133

**Comment by FAETHER** [comment]
Close this, it's fixed in the current version of executable.

**Comment by Uveso** [comment]
@FAETHER
Any information about what caused the bug ? 
Would be nice if i can write something into the first post :)

**Comment by FAETHER** [comment]
Bro, I've mentioned this in slack ~month ago. For those who don't follow slack, is happens when a unit tries to attack another entity and passes the range check, but the entity does not exist anymore, so it's a null pointer exception. There is no way to avid it except just cancel the attack order for the unit. This is probably introduced by faf code changes, cause I saw it happen only in faf. @Uveso

**Comment by Uveso** [comment]
Thank's alot FAETHER.

`I am on slack every day but slack deleted your message ~weeks ago ;)`
